src/bootloader/rollbacks/fiovb.h

```diff
@@ -35,9 +35,21 @@ class FiovbRollback : public Rollback {
   }
 
   void installNotify(const Uptane::Target& target) {
+    std::string version = getVersion(target);
+    if (version.empty()) {
+      return;
```

Comment:
@igoropaniuk can you confirm that this is what you expect?

Reply:
yes, we should not update if we can not obrtain read version txt file.